conda_forge_tick/utils.py

```diff
@@ -394,17 +396,12 @@ def __getitem__(self, name: Any) -> str:
         return f'{self}["{name}"]'
 
 
-def setup_logger(logger: logging.Logger, level: str = "INFO") -> None:
-    """Basic configuration for logging"""
-    logger.setLevel(level.upper())
-    ch = logging.StreamHandler()
-    ch.setLevel(level.upper())
-    ch.setFormatter(
-        logging.Formatter("%(asctime)-15s %(levelname)-8s %(name)s || %(message)s"),
+def setup_logging(level: str = "INFO") -> None:
+    logging.basicConfig(
+        format="%(asctime)-15s %(levelname)-8s %(name)s || %(message)s",
+        level=level.upper(),
     )
-    logger.addHandler(ch)
-    # this prevents duplicate logging messages
-    logger.propagate = False
+    logging.getLogger("urllib3").setLevel(logging.INFO)
```

Comment:
It looks like this change broke the bot

ref: https://github.com/regro/cf-graph-countyfair/pull/19#issuecomment-1979822117

Reply:
Bot is functioning fine. It broke ci tests on that repo which are there only to double check manual changes.

Reply:
That's fair. Wrote that kind of tersely

Am [not seeing this function used in the org]( https://github.com/search?q=org%3Aregro+setup_logger&type=code ). Are we getting an old package?